docs/guides/quickstart-dfp.md

```diff
@@ -0,0 +1,258 @@
+- [bloxone_dfp_service](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/dfp_service)
+
+We use the service ID of the Infra Service block to create the DFP Service.
+
+Add the following code to your main.tf:
+
+````terraform
+resource "bloxone_td_internal_domain_list" "example_list" {
+  name = "example_internal_domain_list"
+  internal_domains = ["example.domain.com"]
+}
+
+# Create the DFP Service
+resource "bloxone_dfp_service" "example" {
+  service_id = bloxone_infra_service.example.id
+
+  # Other optional fields
+  internal_domain_lists = [bloxone_td_internal_domain_list.example_list.id]
+  resolvers_all = [
+    {
+      address = "1.1.1.1"
+      is_fallback = true
+      is_local = false
+      protocols = ["DO53"]
+    }
+  ]
+}
+````
+The `resolvers_all` attribute is used to specify the DNS resolvers for the DFP service.
+
+You can now run `terraform plan` to see what resources will be created.
+
+```shell
+terraform plan
+```
+
+### Creating the Security Policy and Resources associated with it
+In this example, you will use the following resources to create a Custom List, Bypass Code and an External Network.
+
+- [bloxone_td_named_list (Custom List)](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/td_named_list)
+- [bloxone_td_access_code (Bypass Code)](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/td_access_code)
+- [bloxone_td_network_list (External Network)](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/td_network_list)
+
+Add the following to the `main.tf` file:
+
+````terraform
+# Create the Named List
+resource "bloxone_td_named_list" "example" {
+  name = "example_named_list"
+  items_described = [
+    {
+      item        = "tf-domain.com"
+      description = "Example Domain"
+    }
+  ]
+  type = "custom_list"
+}
+
+# Create the Access Code using the Named List
+resource "bloxone_td_access_code" "example" {
+  name       = "example_access_code"
+  activation = timestamp()
+  expiration = timeadd(timestamp(), "24h")
+  rules = [
+    {
+      data = bloxone_td_named_list.example.name,
+      type = bloxone_td_named_list.example.type
+    }
+  ]
+  # Other optional fields
+  description = "Example Access Code"
+}
+
+# Create the Network List
+resource "bloxone_td_network_list" "example" {
+  name  = "example_network_list"
+  items = ["156.2.3.0/24"]
+
+  # Other optional fields
+  description = "Example Network List"
+}
+
+
+````
+The `rules` attribute in the Access code resource is used to specify the Named List created earlier.
+
+You can now run `terraform plan` to see what resources will be created.
+
+```shell
+terraform plan
+```
+
+Finally, you will create the Security Policy that uses the Custom List, Bypass Code and an External Network created earlier.
```

Comment:
If this document is specific to DFP, then I would not like to add content for Bypass Code and External Network List

Reply:
The idea was that , since we are already demonstrating a DFP based Security policy , we demonstrate all the other TD objects associated with it . This has been mentioned in the description too .